packages/gatsby-theme-docs/src/components/video-player-client-side.js

```diff
@@ -0,0 +1,57 @@
+import React, { useRef, useEffect } from 'react';
+import videojs from 'video.js';
+import PropTypes from 'prop-types';
+import 'video.js/dist/video-js.css';
+
+const prepareVideoOptions = (options, videoUrl, autoplay, poster) => {
```

Comment:
1. Can you hide the default caption and (whatever the headphone symbol is) controls?  They seem to appear independent of whether there are captions, or at least I am not seeing any captions when I activate one.   If, then we'll use a single button for english captions on/off (as discussed today, no perspective for non-english content).   Done through passing options to the component tree: https://videojs.com/guides/components/#default-component-tree  (scroll down from there to see the options and their value.  Sometimes it's just passing `false` for a component to remove it)

2. I suggest to hardwire language to english ( https://videojs.com/guides/options/#language ) - it would be confusing if all our UI controls text is english but the player uses browser localized strings.

Reply:
PS: I ran across a nice hack to remove the PiP on devices that don't support it (typically mobiles):  

`{ controlBar: { pictureInPictureToggle: !!document.pictureInPictureEnabled } }`

Reply:
Regarding the first point:
The headphone symbol (audio tracks) and the CC symbol (subtitles) are automatically sown/hidden depending on the presence of multiple audio tracks and CC files. In the example I added, both are present to you get the 2 buttons enabled
I added an additional video (unfortunately I've only found a live stream) that doesn't have additional audio tracks or CC and the 2 buttons are hidden.

The other points are done